app/views/content/salaries-and-benefits.md

```diff
@@ -56,57 +56,52 @@ keywords:
+### Qualified teacher salary
 
 | Area                                     | Minimum | Maximum |
 | -------                                  | -----   | -----   |
-| England (excluding London)     | £30,000 | £41,333 |
-| London fringe                            | £31,350 | £42,689 |
-| Outer London                             | £34,514 | £46,001 |
-| Inner London                             | £36,745 | £47,666 |
-
-Most established teachers will earn more than this maximum by progressing onto the upper pay range for teachers, or by becoming a leading practitioner.
+| England (excluding London)     | £30,000 | £46,525 |
+| London fringe                            | £31,350 | £47,839 |
+| Outer London                             | £34,514 | £51,179 |
+| Inner London                             | £36,745 | £56,959 |
 
-If you’re not sure whether you'll be teaching in inner, outer or the fringe of London, talk to your school.
+### Additional payments
 
-### Upper pay range for teachers
+You may also receive additional payments on top of your regular salary for taking on extra responsibilities. 
 
-If you can demonstrate excellence against all of the teacher standards, you can be put onto the upper pay range for teachers.
+This could be a one-off payment for a specific project, for example, creating a timetable. 
 
-This is for teachers who make a sustained and substantial contribution to their school.
+Or it might be an extra responsibility attached to your job role, for example, being the head of department. 
 
-The upper pay ranges for teachers are:
+Depending on your responsibilities, you could earn up to an extra £15,690. Your school will determine how much you receive. 
 
-| Area                                     | Minimum | Maximum |
-| -------                                  | -----   | -----   |
-| England (excluding London)     | £43,266 | £46,525 |
-| London fringe                            | £44,579 | £47,839 |
-| Outer London                             | £47,592 | £51,179 |
-| Inner London                             | £52,526 | £56,959 |
+## Career progression
 
-### Leading practitioner salaries
+### Leading practitioner salary
 
-If you’re an established and exceptional teacher, and regularly show the highest standards of classroom teaching, you can be put onto a higher pay scale.
+Some schools also have a leading practitioner pay range for teachers with exemplary teaching skills who lead the improvement of teaching in their school.
 
-Although they may not lead departments, leading practitioners coach and mentor other teachers and induct trainees and early career teachers (ECTs).
+What this will involve will depend on your school, but could include:
 
-The teacher pay scales for leading practitioners are:
+* coaching and mentoring other teachers, including trainees and early career teachers
+* undertaking a national professional qualification (NPQ) 
+* supporting your school to produce schemes of work to reduce workload
```

Comment:
@EllieNodder `schemes of work` feels very jargony - I don't know what it means! Is there another way to describe this?